pkg/controller/command/didclient/command.go

```diff
@@ -102,9 +129,29 @@ func (c *Command) CreateDID(rw io.Writer, req io.Reader) command.Error {
 		return command.NewExecuteError(CreateDIDErrorCode, err)
 	}
 
-	command.WriteNillableResponse(rw, m, logger)
+	command.WriteNillableResponse(rw, &CreateDIDResponse{
+		DID:        m,
+		PrivateKey: didPrivateKey ,
+	}, logger)
 
 	logutil.LogDebug(logger, commandName, createDIDCommandMethod, successString)
 
 	return nil
 }
+
+
+func generateECKeyPair() ([]byte, []byte, error){
```

Comment:
try to do like this https://github.com/trustbloc/trustbloc-did-method/blob/master/pkg/vdri/trustbloc/vdri.go#L39

Reply:
will help your unit test

Reply:
yes that did the trick now project coverage is green above 85, patch is still low though.